src/main/java/hudson/plugins/build_timeout/impl/DeadlineTimeOutStrategy.java

```diff
@@ -0,0 +1,146 @@
+package hudson.plugins.build_timeout.impl;
+
+import hudson.Extension;
+import hudson.model.BuildListener;
+import hudson.model.AbstractBuild;
+import hudson.model.Descriptor;
+import hudson.plugins.build_timeout.BuildTimeOutStrategy;
+import hudson.plugins.build_timeout.BuildTimeOutStrategyDescriptor;
+import hudson.util.FormValidation;
+
+import java.io.IOException;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.logging.Logger;
+
+import org.jenkinsci.plugins.tokenmacro.MacroEvaluationException;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.QueryParameter;
+
+/**
+ * If the build reaches <tt>deadlineTime</tt>, it will be terminated.
+ * 
+ * @author Fernando Migu�lez Palomo <fernando.miguelez@gmail.com>
+ */
+public class DeadlineTimeOutStrategy extends BuildTimeOutStrategy {
+
+    public static final int MINIMUM_DEADLINE_TOLERANCE_IN_MINUTES = 1;
+
+    public static final Logger LOG = Logger.getLogger(DeadlineTimeOutStrategy.class.getName());
+
```

Comment:
Right. I will remove it, since I no longer use it. Using listener output instead.